File: antoniohurt-java-blue-material/module-1/09_Introduction_Classes/student-exercise/src/main/java/com/techelevator/person/Person.java
package com.techelevator.person;

public class Person {

	private String firstName;
	private String lastName;
	private int age;

	public String getFullName() {
		return ("John Doe");

	}
	
	public String getFirstName() {
		return firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public int getAge() {
		return age;
	}

	public boolean isAdult() {
		return age >= 18;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public void setAge(int age) {
		this.age = age;
	}
}

File: exercise-solutions/module-1/09_Introduction_Classes/exercise-final/dotnet/PersonExercise/Classes/Person.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TechElevator.Classes
{
    public class Person
    {

        public string FirstName { get; set; }        
        public string LastName { get; set; }        
        public int Age { get; set; }
        

        public string GetFullName()
        {
            return this.FirstName + " " + this.LastName;
        }

        public bool IsAdult()
        {
            return this.Age >= 18;
        }

    }
}
